src/SourceBuild/content/test/Microsoft.DotNet.UnifiedBuild.Tests/Microsoft.DotNet.UnifiedBuild.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project should build with the Arcade SDK but it currently doesn't because of https://github.com/dotnet/dotnet/blob/main/test/Directory.Build.props. Consider moving that file into the smoke tests dir.